Framework/Core/include/Framework/HistogramRegistry.h

```diff
@@ -93,6 +93,7 @@ class HistogramRegistry
   HistPtr add(const HistogramSpec& histSpec);
   HistPtr add(char const* const name, char const* const title, const HistogramConfigSpec& histConfigSpec, bool callSumw2 = false);
   HistPtr add(char const* const name, char const* const title, HistType histType, const std::vector<AxisSpec>& axes, bool callSumw2 = false);
+  HistPtr add(const std::string_view& name, char const* const title, HistType histType, const std::vector<AxisSpec>& axes, bool callSumw2 = false);
```

Comment:
I would drop the reference and const. string_view is cheap.

Reply:
Thanks, done!